WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -587,9 +587,6 @@ local function LayoutDisplayButtons(msg)
         WeakAuras.UpdateDisplayButton(data);
 
         frame.buttonsScroll:AddChild(displayButtons[data.id]);
-        if(WeakAuras.regions[data.id].region.SetStacks) then
```

Comment:
I think this was an old thing to make sure that %s on icon(?) region types would show up when options was open.

Reply:
Yes, but nowadays the whole system works a bit diffrently. This might break a few things, but then I have to fix them properly